### ISSUE TITLE
Comment starting with "Date" is parsed... strangely

### ISSUE THREAD

**Issue by mat813** [sequelize/sequelize-auto#471]
I have a table in a PostgreSQL database with this comment: 

```sql
COMMENT ON COLUMN ban.bans.date_der_maj IS 'Date de la dernière mise à jour de l’adresse.';
```

And the generated model contains:

```ts
        date_der_maj: {
          type: DataTypes.DATE,
          allowNull: false,
          comment: DataTypes.DATE,
        },
```

**Comment by steveschmitt** [comment]
Strange indeed!  I'll take a look.

**Comment by steveschmitt** [comment]
Fixed in 0.7.6
